src/etc/rc.syshook.d/monitor/20-recover

```diff
@@ -35,13 +35,32 @@ require_once 'interfaces.inc';
 $gwnames = [];
 $affected_gateways = !empty($argv[1]) ? explode(',', $argv[1]) : [];
 
+$metered_found_prios = ['inet' => 256, 'inet6' => 256];
+$metered_gws = ['inet' => [], 'inet6' => []];
 foreach (return_gateways_status() as $status) {
-    if ($status['status'] == 'down') {
+    if ($status['status'] != 'none') {
```

Comment:
this is definitely not going to take on a large fan base assuming this wasn't broken